### PR TITLE
tomlq: init at 0.1.6

### PR DESCRIPTION
**Pull request by kinzokudev**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/to/tomlq/package.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    repo = "tomlq";
+    rev = version;
+    hash = "sha256-g8xjz8qCTiulTwcEbLTHYldw4PI+4ZfCOMJs+J6L1C4=";
+  };
+
+  cargoHash = "sha256-/cepTVJoBM1LYZkFpH9UCvE74cSszHDaeThsZksQ1P8=";
+
+  meta = with lib; {
+    description = "A tool for getting data from TOML files on the command line";
+    homepage = "https://github.com/cryptaliagy/tomlq";
```
[the remark this answers is not in the document]
Comment (kinzoku-dev) [reply pkgs/by-name/to/tomlq/package.nix:21]: What do you mean by this?

File: pkgs/by-name/to/tomlq/package.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tomlq";
+  version = "0.1.6";
+
+  src = fetchFromGitHub {
+    owner = "cryptaliagy";
+    repo = "tomlq";
+    rev = version;
```
[the remark this answers is not in the document]
Comment (kinzoku-dev) [reply pkgs/by-name/to/tomlq/package.nix:13]: is this really needed? it seemed to work completely fine without it

File: pkgs/by-name/to/tomlq/package.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aleksanaa) [reply pkgs/by-name/to/tomlq/package.nix:21]: `changelog = "https://github.com/cryptaliagy/tomlq/releases/tag/${version}";`

File: pkgs/by-name/to/tomlq/package.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  src = fetchFromGitHub {
+    owner = "cryptaliagy";
+    repo = "tomlq";
+    rev = "refs/tags/${version}";
+    hash = "sha256-g8xjz8qCTiulTwcEbLTHYldw4PI+4ZfCOMJs+J6L1C4=";
+  };
+
+  cargoHash = "sha256-/cepTVJoBM1LYZkFpH9UCvE74cSszHDaeThsZksQ1P8=";
+
+  meta = with lib; {
```
Comment (Aleksanaa) [inline pkgs/by-name/to/tomlq/package.nix:19]:
```suggestion
  meta = {
```

File: pkgs/by-name/to/tomlq/package.nix [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    description = "A tool for getting data from TOML files on the command line";
```
Comment (Aleksanaa) [inline pkgs/by-name/to/tomlq/package.nix:20]:
```suggestion
    description = "Tool for getting data from TOML files on the command line";
```

File: pkgs/by-name/to/tomlq/package.nix (hunk 3 above)
Comment (kinzoku-dev) [reply pkgs/by-name/to/tomlq/package.nix:19]: Go ahead and also change the `with maintainers;` part to `with lib.maintainers;` and i'll commit this

File: pkgs/by-name/to/tomlq/package.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (kinzoku-dev) [reply pkgs/by-name/to/tomlq/package.nix:21]: > `changelog = "https://github.com/cryptaliagy/tomlq/releases/tag/${version}";`

So I just add this in the `meta` attrset?

File: pkgs/by-name/to/tomlq/package.nix (hunk 3 above)
Comment (Aleksanaa) [reply pkgs/by-name/to/tomlq/package.nix:19]: There's web editor in `Files changed` overview. Doing squash merge is just a grace, and originally according to the guidelines we were not supposed to do this.

Comment (kinzoku-dev) [reply pkgs/by-name/to/tomlq/package.nix:19]: Oh okay, thanks!

**Comment by kinzoku-dev** [comment]
> Squash them into one commit please.

Is there a way to do this in the GitHub web UI?

**Comment by kinzoku-dev** [comment]
Ok, done

**Comment by ardishko** [comment]
LGTM

**Review by ardishko — APPROVED** [review]

**Review by h7x4 — APPROVED** [review]

**Pull request merged**